Resources/config/schema/swiftmailer-1.0.xsd

```diff
@@ -66,6 +66,10 @@
     </xsd:restriction>
   </xsd:simpleType>
 
+  <xsd:simpleType name="delivery-address">
```

Comment:
Build fails due to this update. Won't fail after it's published on http://symfony.com/schema/dic/swiftmailer/swiftmailer-1.0.xsd.